src/index.html

```diff
@@ -15,8 +15,23 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html"><img  src="images/logo.png" alt="Moyo" > </img></a>
```

Comment:
The `alt` attribute for the `img` tag should describe the image content. Please, provide a more descriptive `alt` attribute for better SEO and accessibility. For example, 'Moyo logo' instead of just 'Moyo'.